src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java

```diff
@@ -212,17 +224,19 @@ public static void checkVaccinations(String key, Map<String, Object> body) {
      */
     public static void checkIllnesses(String key, Map<String, Object> body) {
         checkDateFormat(key);
-        if (body.size() != 4 || !body.containsKey("endDateTime") || !body.containsKey("type")
-            || !body.containsKey("description") || !body.containsKey("severity")) {
-            throw new IllegalArgumentException("Request body does not have a correct format");
+        String type = "type";
+        String severity = "severity";
+        if (body.size() != 4 || !body.containsKey(END_DATE_TIME) || !body.containsKey(type)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean expression complexity is 4 (max allowed is 3).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)